conda-forge.yml

```
@@ -1,10 +1,10 @@
 build_platform:
-  linux_aarch64: linux_64
-  linux_ppc64le: linux_64
-  osx_arm64: osx_64
+  linux_ppc64le: linux_ppc64le  # tests fail when emulating power from x86
```

Think this can just be `default` since we build natively there

```suggestion
  linux_ppc64le: default
```

---

I explicitly choose `linux_ppc64le` because we know that the build doesn't work when emulated. The default could be changed to `linux_64` like it was for linux_aarch64, then this build would break again.

---

Also, It only renders with `linux_ppc64le`. I cannot use the `default` key here; it causes the following error:
```
  File "/home/dching/miniconda3/envs/forge/lib/python3.9/site-packages/conda_smithy/configure_feedstock.py", line 947, in _get_platforms_of_provider
    build_platform, build_arch = build_platform_arch.split("_")
ValueError: not enough values to unpack (expected 2, got 1)

```